lookout/core/tests/test_metrics.py

```diff
@@ -0,0 +1,186 @@
+import unittest
```

Comment:
I need a multithreading test, as we discussed

Reply:
I'm not forgetting about this, but I will do it tomorrow morning.

Reply:
Is this still needed?